<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>初始化</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="2">
               <widget class="QPushButton" name="pushButton_insert_ok">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="text">
                 <string>插入sheet</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="text">
                 <string>选择sheet</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="pushButton_select_ok">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_select_sheet">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QComboBox" name="comboBox_sheet">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <item>
                   <property name="text">
                    <string>Sheet</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox_sheet_num">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>13</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>15</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>17</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>18</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>操作日志</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTextEdit" name="textEdit_optlog">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>操作状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>操作状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_status_result">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="uploadProgressBar">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>数据录入</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_startid">
               <property name="font">
                <font>
                 <family>Aharoni</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="font">
                <font>
                 <family>Aharoni</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>录入ID</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="font">
                <font>
                 <family>Aharoni</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>需求个数</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_need_num">
               <property name="font">
                <font>
                 <family>Aharoni</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>产品型号</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>产品大小</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>产品颜色</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>电池类型</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>防水等级</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>输入电压</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>电池容量</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>输入电流</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>日期时间</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="lineEdit_i">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_v">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEdit_cap">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_type">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_size">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_color">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBox_bat">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_water">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_reset">
             <property name="minimumSize">
              <size>
               <width>245</width>
               <height>350</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_write">
             <property name="minimumSize">
              <size>
               <width>245</width>
               <height>350</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>录入</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_create"/>
    <addaction name="action_open"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_create">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
